index.bs

```diff
@@ -3036,6 +3036,14 @@ optionally evidence of [=user consent=] to a specific transaction.
         The [=client=] and [=authenticator=] make a best-effort to create a credential of the most preferred type possible.
         If none of the listed types can be created, the {{CredentialsContainer/create()}} operation fails.
 
+        [=[RP]=]'s that wish to support a wide range of [=authenticators=] SHOULD define entries for the following discrete list of signature algorithms:
```

Comment:
`[=[RP]=]'s` would render as `Relying Party's`. Use the `[RPS]` macro instead for the plural.

```suggestion
        [=[RPS]=] that wish to support a wide range of [=authenticators=] SHOULD define entries for the following discrete list of signature algorithms:
```

Reply:
So...many...macros...but an easy enough change so I went ahead with it.